Sources/PushNotifications.swift

```diff
@@ -143,7 +144,7 @@ import Foundation
      */
     /// - Tag: setSubscriptions
     @objc public func setSubscriptions(interests: Array<String>, completion: @escaping () -> Void = {}) throws {
-        if let invalidInterests = self.validateInterestNames(interests) {
+        if let invalidInterests = self.validateInterestNames(interests), invalidInterests.count > 0 {
```

Comment:
Why is this `invalidInterests.count` now present?

Should you have a test for this?

Reply:
If we call method `setSubscriptions` and pass interests array with only one valid interest, for example `["hello"]` it would throw an exception `MultipleInvalidInterestsError`.

Reply:
Write a regression test then! :)